meta-dts-distro/recipes-bsp/coreboot-utils/futil_git.bb

```diff
@@ -12,6 +12,8 @@ DEPENDS += " \
     flashrom \
 "
 
+PV = "1.0+git${SRCPV}"
```

Comment:
Will `${SRCPV}` append whole commit hash, that is 40 characters? If yes then maybe shorten it to ~10 characters?
This is just suggestion.

Reply:
```shell
bash-5.2# futility version
v0.0.2897-dc68f9f1-dirty 2024-06-19 08:06:24
```

Reply:
This no longer applies I think 68f9926e5e6a51d61325ef1f0cd8e6a3cba7c8e5

Reply:
This recipe doesn't include `coreboot-utils.inc` so you need to set it manually it's `Dasharo/vboot` not `Dasharo/coreboot`

Reply:
`PV = "1.0+git${SRCPV}"` results in `futil core2_64 1.0+git0+dc68f9f1b5`

Reply:
I changed it back https://github.com/Dasharo/meta-dts/compare/68f9926e5e6a51d61325ef1f0cd8e6a3cba7c8e5..e7a278a16a3415a585b227982681d5751015e603